pkg/cmd/events/io.go

```diff
@@ -33,7 +33,22 @@ func PrintEvents(writer io.Writer, events []*corev1.Event) error {
 		message = strings.Replace(message, `"""`, `"`, -1)
 		message = strings.Replace(message, "\t", "\t", -1)
 
-		if _, err := fmt.Fprintf(writer, "%s (%d) %q %s %s\n", event.FirstTimestamp.Format("15:04:05"), event.Count, event.InvolvedObject.Namespace, event.Reason, message); err != nil {
+		countMessage := fmt.Sprintf("%d", event.Count)
+		if event.Count > 1 {
+			// eventDuration represents the time between first and last event observed
```

Comment:
@sttts this is also important... to know the duration of the event (firstTimestamp <-> lastTimestamp)